src/components/Footer/FooterExtendedNavList/FooterExtendedNavList.stories.tsx

```diff
@@ -0,0 +1,33 @@
+/*  eslint-disable jsx-a11y/anchor-is-valid, react/jsx-key */
+import React from 'react'
+
+import { FooterExtendedNavList } from './FooterExtendedNavList'
+
+export default {
+  title: 'Footer/FooterExtendedNavList',
+  parameters: {
+    info: `
+     Display grouped nav items in an extended nav. Used in  "big" USWDS 2.0 Footer component.
+    
+    Source: https://designsystem.digital.gov/components/form-controls/#footer
+    `,
+  },
+}
+
+export const Default = (): React.ReactElement => (
+  <div className="usa-footer--big ">
```

Comment:
It doesn't feel great React-wise to have to wrap a component to get it to display mobile styles. But there is a dependency in the CSS here.

Reply:
I ran into this in the header, and I couldn't find a way around it either. I agree, it definitely felt a little dirty...

Reply:
Agreed, but ultimately this is a shortcoming of the USWDS CSS so IMO all we can do is work with it :( IMO as long as we document clearly that this component should only ever be used within the "big" Footer, that's good enough